### PR TITLE
flappy bird [hacktoberfest]

### PR DESCRIPTION
**Pull request by ravi0900**
added flappy bird game template
please add me as assignee and add hacktoberfest and hacktoberfest-accepted label
please merge this pr
thank you.

### CODE REVIEW

File: flappy bird/flappybird.html [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+            <div id="DebugInfo" class="noSelect"></div>
+			<div id="Birdy" style=" background-position-x: 400%;">
+			</div>
+            <div id="PauseButton" class="noSelect">
+                <span class="glyphicon glyphicon-pause"></span>
+            </div>
+		</div>
+		
+        <!-- jQuery from Google CDN -->
+		<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
```
[the remark this answers is not in the document]
Comment (Diptenusarkar) [reply flappy bird/flappybird.html:227]:
``` suggestion
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
```

**Comment by ravi0900** [comment]
@Diptenusarkar added suggested changes

**Review by Diptenusarkar — APPROVED** [review]
